src/tracer/TraceExecutor.cc

```diff
@@ -205,7 +205,8 @@ std::tuple<TraceAction, int> TraceExecutor::handleTraceeSignal(
         // Mask is shifted by one because the lowest bit of SigCgt mask
         // corresponds to signal 1, not 0.
         uint64_t caughtSignals =
-                procfs::readProcFS(tracee.getPid(), procfs::Field::SIG_CGT) << 1;
+                procfs::readProcFS(tracee.getPid(), procfs::Field::SIG_CGT)
+                << 1;
```

Comment:
```suggestion
                procfs::readProcFS(tracee.getPid(), procfs::Field::SIG_CGT) << 1;
```
Can we keep this line unchanged?

Reply:
I think that it's better to let clang-format handle this line now than to require future contributors to run `git restore ../src/perf/PerfListener.cc` after every `make clang-format`.

Reply:
Okay, I am not against formatting the codebase to keep it consistent. I am just against doing that in incremental fashion combined with actual changes. I would rather format whole code and commit it as separate commit.
a) This line does not match commit title "Add support for heterogenous cpus to PerfListener", it really makes existing code conform to the formatting guidelines.
b) Making it a separate commit allows easier history analysis with `git blame` and other tools, where we can ignore some no-op commits.
c) I thinks smaller commits not only make code history more visible, but also makes rebasing easier in case someone maintains their forks of this code.

It's just it. I am totally for having codebase well formatted and better dev experience, just not everything at once in a single commit.

Reply:
I wasn't sure as to whether adding a one/two-line commit was better than keeping the change in the main commit, thanks for the convincing arguments!
I separated the formatting commit.

Reply:
Perfect, Thank You!